ehrapy/tools/_sa.py

```diff
@@ -184,9 +184,17 @@ def kmf(
     return kmf
 
 
-def calculate_nested_f_statistic(small_model, big_model):
-    # Using code from https://stackoverflow.com/questions/27328623/anova-test-for-glm-in-python/60769343#60769343
-    """Given two fitted GLMs, the larger of which contains the parameter space of the smaller, return the P value corresponding to the larger model adding explanatory power"""
+def calculate_nested_f_statistic(small_model: GLMResultsWrapper, big_model: GLMResultsWrapper) -> float:
+    """Given two fitted GLMs, the larger of which contains the parameter space of the smaller, return the P value corresponding to the larger model adding explanatory power
+    See https://stackoverflow.com/questions/27328623/anova-test-for-glm-in-python/60769343#60769343
```

Comment:
What Google docstrings always expect is:

```
"""one sentence description.

more details

Args.
blbalbalba
"""
```

So the proper way to format this would be 
```
"""Given two fitted GLMs,..explanatory power.

See link bla
"""
```

Reply:
You might have to tab in the additional detail sentences. Not sure anymore :) See other functions

Reply:
> What Google docstrings always expect is:
> 
> ```
> """one sentence description.
> 
> more details
> 
> Args.
> blbalbalba
> """
> ```
> 
> So the proper way to format this would be
> 
> ```
> """Given two fitted GLMs,..explanatory power.
> 
> See link bla
> """
> ```

Got it! Thanks for pointing out this! I just installed a autodocstring extension to help this : )